HueTests/iOS/UIColorTests.swift

```diff
@@ -37,61 +37,61 @@ class UIColorTests: XCTestCase {
   }
 
   func testToHexWithoutPrefix() {
-    let white = UIColor.whiteColor()
-    let black = UIColor.blackColor()
-    let red = UIColor.redColor()
-    let blue = UIColor.blueColor()
-    let green = UIColor.greenColor()
-    let yellow = UIColor.yellowColor()
+    let white = UIColor.white
+    let black = UIColor.black
+    let red = UIColor.red
+    let blue = UIColor.blue
+    let green = UIColor.green
+    let yellow = UIColor.yellow
 
-    XCTAssertEqual(white.hex(withPrefix: false), "FFFFFF")
-    XCTAssertEqual(black.hex(withPrefix: false), "000000")
-    XCTAssertEqual(red.hex(withPrefix: false), "FF0000")
-    XCTAssertEqual(blue.hex(withPrefix: false), "0000FF")
-    XCTAssertEqual(green.hex(withPrefix: false), "00FF00")
-    XCTAssertEqual(yellow.hex(withPrefix: false), "FFFF00")
+    XCTAssertEqual(white.hex(false), "FFFFFF")
+    XCTAssertEqual(black.hex(false), "000000")
+    XCTAssertEqual(red.hex(false), "FF0000")
+    XCTAssertEqual(blue.hex(false), "0000FF")
+    XCTAssertEqual(green.hex(false), "00FF00")
+    XCTAssertEqual(yellow.hex(false), "FFFF00")
   }
 
   func testAlpha() {
-    let yellowWithAlpha = UIColor.hex("#FFFF00").alpha(0.5)
+    let yellowWithAlpha = UIColor(hex: "#FFFF00").alpha(0.5)
 
-    XCTAssertEqual(yellowWithAlpha, UIColor(red: 255, green: 255, blue: 0, alpha: 1.0).colorWithAlphaComponent(0.5))
+    XCTAssertEqual(yellowWithAlpha, UIColor(red: 1, green: 1, blue: 0, alpha: 1.0).withAlphaComponent(0.5))
   }
 
   func testGradient() {
-    let testGradient = [UIColor.blackColor(), UIColor.orangeColor()].gradient()
+    let testGradient = [UIColor.black, UIColor.orange].gradient()
 
-    XCTAssertTrue(testGradient.isKindOfClass(CAGradientLayer))
+    XCTAssertTrue(testGradient as Any is CAGradientLayer)
```

Comment:
Nope, because it will complains that the type is valid and the test will always succeed. We should either remove the test or use it as is.